packages/util/src/account.ts

```diff
@@ -122,7 +122,12 @@ export class Account {
    * "An account is considered empty when it has no code and zero nonce and zero balance."
    */
   isEmpty(): boolean {
-    return this.balance === _0n && this.nonce === _0n && this.codeHash.equals(KECCAK256_NULL)
+    return (
+      this.balance === _0n &&
+      this.nonce === _0n &&
+      this.codeHash.equals(KECCAK256_NULL) &&
+      this.storageRoot.equals(KECCAK256_RLP)
```

Comment:
I remember changing this in the past as well too. This breaks tests. I think it is related to gas costs, if all fields are zero except `storageRoot` then the account, for EVM purposes, is still empty. 

If you remove this line, `npm run test:state -- --fork=London` changes from `fail  1778` to `fail  186`

Reply:
Have reverted.